src/types/identity.rs

```diff
@@ -0,0 +1,18 @@
+use std::collections::HashMap;
+
+use pyo3::{pyclass, pymethods};
+
+#[pyclass]
+#[derive(Debug, Clone)]
+pub struct Identity {
+    #[pyo3(get, set)]
+    claims: HashMap<String, String>,
+}
```

Comment:
Just two questions regarding `Identity`
 - should we keep it as a TypedDict or a DataClass
 - And is using the name `Identity` a commonly used practice?

Reply:
- Regarding the TypedDict, I explained in an other comment that I would prefer keeping it as a dataclass.
- For `Identity`, this is actually an inspiration I took from `GuardPost` (see [here](https://github.com/Neoteroi/GuardPost/blob/main/guardpost/authentication.py#L11)). I used this project and `Blacksheep` to design the system for Robyn :)